File: Client/Pikachu/PikachuLib/Communication/Models/Usermanagement/UserResponse.cs
using Newtonsoft.Json;

namespace PikachuLib.Communication.Models.Usermanagement
{
    /// <summary>
    /// Repräsentation eines User Response
    /// </summary>
    public class UserResponse
    {
        #region properties
        [JsonProperty("id")]
        public int Id { get; set; }

        [JsonProperty("username")]
        public string Username { get; set;}

        [JsonProperty("password")]
        public string Password { get; set;}

        [JsonProperty("prename")]
        public string Prename { get; set;}

        [JsonProperty("name")]
        public string Name { get; set;}

        [JsonProperty("admin")]
        public bool IsAdmin { get; set;}

        [JsonProperty("active")]
        public bool IsActive { get; set;}
        #endregion
    }
}

File: Server/relaxo/src/main/java/de/teamrocket/relaxo/rest/models/usermanagement/response/UserResponse.java
package de.teamrocket.relaxo.rest.models.usermanagement.response;

import de.teamrocket.relaxo.models.usermanagement.User;

/**
 * Created by nakih on 09.01.15.
 */
public class UserResponse {

    private final String username;
    private final String name;
    private final String prename;
    private final boolean isAdmin;
    private final boolean isActive;
    private final int id;

    public UserResponse(User user) {
        this.name = user.getName();
        this.prename = user.getPrename();
        this.username = user.getUsername();
        this.isAdmin = user.isAdmin();
        this.isActive = user.isActive();
        this.id = user.getId();
    }


    public String getUsername() {
        return username;
    }

    public String getName() {
        return name;
    }

    public String getPrename() {
        return prename;
    }

    public boolean isAdmin() {
        return isAdmin;
    }

    public boolean isActive() {
        return isActive;
    }

    public int getId() {
        return id;
    }
}
